auto_install/install.sh

```diff
@@ -29,26 +28,34 @@ ovpnUserGroup="openvpn:openvpn"
 
 ######## PKG Vars ########
 PKG_MANAGER="apt-get"
-### FIXME: quoting UPDATE_PKG_CACHE and PKG_INSTALL hangs the script, shellcheck SC2086
+### FIXME: quoting UPDATE_PKG_CACHE and PKG_INSTALL hangs the script,
+### shellcheck SC2086
 UPDATE_PKG_CACHE="${PKG_MANAGER} update -y"
 PKG_INSTALL="${PKG_MANAGER} --yes --no-install-recommends install"
-PKG_COUNT="${PKG_MANAGER} -s -o Debug::NoLocking=true upgrade | grep -c ^Inst || true"
+PKG_COUNT="${PKG_MANAGER} -s -o Debug::NoLocking=true upgrade | "
+PKG_COUNT="${PKG_COUNT} grep -c ^Inst || true"
```

Comment:
Changed to fit 80 character limit but I find this and similar multiline variable definitions confusing to read and too verbose.